mappings/loc.py

```diff
@@ -51,41 +59,49 @@ def applyFormatting(self):
     #Identifier/SourceID Formatting
     def formatIdentifierSourceID(self, itemList):
         newIdentifier = itemList
-        newIdentifier['call_number'][0] = f'{newIdentifier["call_number"][0]}|call_number'
         lccnNumber = self.record.identifiers[0][0]  #lccnNumber comes in as an array and we need the string inside the array
-        callNumber = newIdentifier['call_number'][0].strip(' ')
         sourceID = lccnNumber
+        if 'call_number' in newIdentifier.keys():
+            newIdentifier['call_number'][0] = f'{newIdentifier["call_number"][0]}|call_number'
+            callNumber = newIdentifier['call_number'][0].strip(' ')
+        else: 
+            callNumber = None
         return (lccnNumber, callNumber, sourceID)
     
     #Publisher/Spatial Formatting
     def formatPubSpatial(self, itemList):
```

Comment:
I think maybe a docstring explaining the input / output a bit (doesn't need to be super detailed) would be helpful, since there's a lot of logic in this method!

Reply:
Sounds good.